src/SlashCommands.tsx

```diff
@@ -1070,6 +1070,28 @@ export const Commands = [
         category: CommandCategories.advanced,
         renderingTypes: [TimelineRenderingType.Room],
     }),
+    new Command({
+        command: 'remakeolm',
+        description: _td('Developer command: Discards the current outbound group session and sets up new Olm sessions'),
+        isEnabled: () => {
+            return SettingsStore.getValue("developerMode");
+        },
+        runFn: (roomId) => {
+            try {
+                const room = MatrixClientPeg.get().getRoom(roomId);
+
+                MatrixClientPeg.get().forceDiscardSession(roomId);
+
+                // noinspection JSIgnoredPromiseFromCall
+                MatrixClientPeg.get().crypto.ensureOlmSessionsForUsers(room.getMembers().map(m => m.userId), true);
+            } catch (e) {
+                return reject(e.message);
+            }
+            return success();
+        },
+        category: CommandCategories.advanced,
+        renderingTypes: [TimelineRenderingType.Room],
+    }),
```

Comment:
This feels a bit duplicative of `/discardsession` and then merely typing into the composer to trigger the megolm setup

Reply:
The distinction is the olm sessions.